web/src/components/tasks/business-formation/NexusFormationFlow.tsx

```diff
@@ -25,10 +25,9 @@ export const NexusFormationFlow = (): ReactElement => {
 
   useMountEffect(() => {
     if (!business) return;
-    if (
-      business.formationData.lastVisitedPageIndex > BusinessFormationStepsConfiguration.length - 1 ||
-      business.formationData.lastVisitedPageIndex < 0
-    ) {
+    const lastVisitedPageIndex = business.formationData.lastVisitedPageIndex;
```

Comment:
I was thinking `const { lastVisitedPageIndex } = business.formationData` for destructuring`lastVisitedPageIndex`.